src/site/stages/build/process-cms-exports/transformers/node-story_listing.js

```diff
@@ -10,26 +11,52 @@ const transform = entity => ({
   title: getDrupalValue(entity.title),
   created: utcToEpochTime(getDrupalValue(entity.created)),
   changed: utcToEpochTime(getDrupalValue(entity.changed)),
+  entityPublished: isPublished(getDrupalValue(entity.moderationState)),
```

Comment:
So the "right' way to determine if an entity is published is actually to check the `status[0].value === true`. We didn't learn this until after we wrote that `isPublished` function and used the moderation state elsewhere.

cc @swirtSJW to confirm

Reply:
👀  Does this mean we should update the `isPublished` function?

Reply:
Yeah, I think that makes sense. :+1:

Reply:
I can do that in a separate PR since we'll have to update all the instances of `isPublished` in the transformers.